scripts/genEventsYoutubePlayerBookmarklet.js

```diff
@@ -0,0 +1,147 @@
+javascript:(function() {
+    const timestamps = [];
+    const durations = []
+    const overlay = createOverlay();
+    document.body.appendChild(overlay);
+
+    document.addEventListener('keydown', handleKeyPress);
+
+    function createOverlay() {
+        const overlay = document.createElement('div');
+        overlay.style.position = 'fixed';
+        overlay.style.top = '50px';
+        overlay.style.right = '0';
+        overlay.style.backgroundColor = 'rgba(255,255,255,0.33)';
+        overlay.style.color = 'white';
+        overlay.style.padding = '10px';
+        overlay.style.zIndex = '1000';
+        return overlay;
+    }
+
+    function handleKeyPress(event) {
+        if (event.key === '=') {
+            captureTimestamp();
+        } else if (event.key === 'Backspace') {
+            removeNearestTimestamp();
+        } else if (event.key === '+') {
+            endTimestamp();
+        }
+    }
+
+    function captureTimestamp() {
+        const currentTime = getCurrentTime();
+        if (currentTime !== null) {
```

Comment:
this works well

if we notice any browser-related oddities, it may be worth changing the prodicate from a null check to a type check for the positive case e.g. 

if (typeof(value) === "number")